crates/bevy_reflect/src/lib.rs

```diff
@@ -881,7 +862,7 @@ mod tests {
 
         let reflected: &dyn Reflect = &test;
         let expected = r#"
-bevy_reflect::tests::should_reflect_debug::Test {
+bevy_reflect::tests::Test {
```

Comment:
Hm, I guess this means we could potentially break this system by doing:

```rust
mod foo {
  #[derive(TypeName)]
  struct Foo;

  fn my_func() {
    #[derive(TypeName)]
    struct Foo;
  }
}
```

Probably not a big deal, though, as that's a very specific circumstance.